### ISSUE TITLE
Error after first epoch while traning 

### ISSUE THREAD

**Issue by iemskl** [cocktailpeanut/fluxgym#154]
An error occurs during training after the first training epoch. Then the error repeats and the training stops. What could be the problem?
```
File "/app/fluxgym/app.py", line 405, in gen_sh 
if len(sample_prompts) > 0 and sample_every_n_steps > 0:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

**Comment by cocktailpeanut** [comment]
did you delete the `sample every n steps` field or the `sample prompts` field before running the training? And if so, which one?

**Comment by iemskl** [comment]
@cocktailpeanut I sample the picture after each epoch depens image number . And.. oh.. zero `Sample Image Prompts` i need to write something.. yes..  thanks))

**Comment by iemskl** [comment]
No, it didn't help.  The `Sample Image Prompts` are the same as the `Trigger word`. Training still stops after 1 epoch with logs `Terminating process <Popen: returncode: None args: ['bash "/app/fluxgym/outputs/v6/train.sh"']>`

**Comment by iemskl** [comment]
Training on rtx4090 , 28 images, resize 1024, network_dim = 16
